### PR TITLE
bt-397: Add transition to elements for Onboarding Flow

### LINKED ISSUE #397: Add transition to elements for Onboarding Flow
When the user hovers over the elements there should be a visual effect
Also, it is necessary to add a transition for states updates
https://developer.mozilla.org/en-US/docs/Web/CSS/transition

### PR DESCRIPTION
**Pull request by Monyster**
Closes #397 

This blue line changes with the transition at each step
image

### CODE REVIEW

File: frontend/src/bundles/common/components/select/styles.module.scss [hunk 1]
```diff
@@ -39,9 +46,9 @@
     background-color: transparent;
 }
 
-.container .input:hover {
-    border-color: var(--botticelli);
-}
+// .container .input:hover {
+//     border-color: var(--st-tropaz);
+// }
```
Comment (MaksymGavatuk) [inline frontend/src/bundles/common/components/select/styles.module.scss]: Remove comments if we don't need this

**Review by MaksymGavatuk — CHANGES_REQUESTED** [review]

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
in the design blue line in one place. we just need to add a transition on elements state (hover, error, etc.)

**Comment by Monyster** [comment]
@nikita-remeslov I can remove it, but as I see on other sites, it should be a moving underline or progress bar.
I can change it to the progress bar style.

If you look at the design, you'll see that this is not a decorative underline for the step name.
image

I suggest to do as shown next
image
OR
image

File: frontend/src/bundles/talent-onboarding/constants/constants.ts [hunk 2]
```diff
@@ -3,6 +3,10 @@ import { StepsRoute } from '~/bundles/talent-onboarding/enums/enums.js';
 const FIRST_ELEMENT = 0;
 const LAST_TWO_CHARS = -2;
 const STEP_ONE = 1;
+const STEP_TWO = 2;
+const STEP_THREE = 3;
+const STEP_FOUR = 4;
+const STEP_FIVE = 5;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/talent-onboarding/constants/constants.ts]: should be enum

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/talent-onboarding/constants/constants.ts [hunk 3]
```diff
@@ -2,7 +2,14 @@ import { StepsRoute } from '~/bundles/talent-onboarding/enums/enums.js';
...
 const FIRST_ELEMENT = 0;
 const LAST_TWO_CHARS = -2;
-const STEP_ONE = 1;
+const StepsList = {
+    ONE: 1,
+    TWO: 2,
+    THREE: 3,
+    FOUR: 4,
+    FIVE: 5,
+};
```
Comment (nikita-remeslov) [inline frontend/src/bundles/talent-onboarding/constants/constants.ts]: check how we create enums

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/talent-onboarding/constants/constants.ts (hunk 3 above)
Comment (Monyster) [reply frontend/src/bundles/talent-onboarding/constants/constants.ts]: as const

**Comment by nikita-remeslov** [comment]
conflicts

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**